pkg/operator/operator.go

```diff
@@ -125,14 +115,6 @@ func NewOperator() (context.Context, *Operator) {
 		debug.SetMemoryLimit(newLimit)
 	}
 
-	// Webhook
```

Comment:
Do we need L116 anymore if we don't use knative anymore? Maybe this gets taken care of in a different PR to fully remove?